src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -238,6 +238,13 @@ public override AbstractFormattingRule WithOptions(AnalyzerConfigOptions options
                 {
                     return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
                 }
+
+                // [Attribute1]$$ [Attribute2]
+                if (currentToken.IsKind(SyntaxKind.OpenBracketToken) &&
+                    currentToken.Parent.IsKind(SyntaxKind.AttributeList))
```

Comment:
this case seems weird given the if check above this.  we already checked if the owner was a member, and if so we force a space.  it seems like we'd want to do the multiple-attribute check *prior* to doing the member check, otherwise we would not hit this for `[Foo]  [Bar] void SomeMember()`, right?

Reply:
For methods, we introduce newlines between attributes.
But for fields, the order here matters. If I swap the order, then `FormatMultipleAttributeOnSameLineAsField1` is affected (it expects  `[Attr3] [Attr4] int i;` with 1 space.
So I think we should keep the order as-is.

Reply:
can you comment teh code explicitly to indicate this is intentional.  note if we want fields to keep thigns on a single line, i'd far prefer that we do that explicitly.  Note: for fields, i would expect no space between attributes as well :(

tagging @sharwell for thoughts here.